extensions/http-curl/tests/HTTPHandlers.h

```diff
@@ -445,4 +445,49 @@ class HeartbeatHandler : public CivetHandler {
   bool isSecure;
 };
 
+class InvokeHTTPResponseOKHandler : public CivetHandler {
+public:
+  bool handlePost(CivetServer *, struct mg_connection *conn) {
+    std::stringstream headers;
+    headers << "HTTP/1.1 201 OK\r\nContent-Type: text/plain\r\nContent-Length: 0\r\nConnection: close\r\n\r\n";
```

Comment:
I don't think stringstream makes sense in case we only put one string to it. 
Can we simply pass the hardcoded cstr to the mg_printf?

Reply:
Absolutely. stringstream is unnecessary here.